cluster-setup/base/performance/operator_subscription.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: performance-addon-operator
   namespace: openshift-performance-addon
 spec:
-  channel: 4.5.0
+  channel: 4.5
```

Comment:
Why? Please keep the old channel name. It is just a name and needs to stay compatible with existing subscriptions.

Reply:
IMHO until we will have official u/s release we should use this chance to align channels names